website/backend/settings.py

```diff
@@ -128,15 +126,7 @@
 # Database
 # https://docs.djangoproject.com/en/3.1/ref/settings/#databases
 
-DATABASES = {
-    "default": dj_database_url.config(
-        # Default values for DATABASE_URI are for the development environment
-        default=env(
-            "DATABASE_URI", default="postgresql://user:password@dbHost:5432/database"
-        )
-    )
-}
-
+DATABASES = {"default": dj_database_url.config(default=env("DATABASE_URI"))}
```

Comment:
this is great and advisable to utilise ENV variables instead of direct input of these values in code. Thanks @123MwanjeMike